system-tests/005/__snapshots__/005.test.js.snap

```diff
@@ -11765,169 +11765,169 @@ Array [
         "text": "Expected line length to be no more than 20 characters (max-line-length)",
       },
       Object {
-        "column": 5,
-        "line": 146,
+        "column": 25,
+        "line": 145,
```

Comment:
@ntwb Yeah it's a funny one - the incorrect column numbers can compound and spill over into incorrect line numbers too. I added an extra example to the bottom of the issue that demonstrate.